operate/importer/src/main/java/io/camunda/operate/zeebeimport/ImportPositionHolder.java

```diff
@@ -127,18 +128,9 @@ public void recordLatestLoadedPosition(final ImportPositionEntity lastProcessedP
           // update only import fields (not post import)
           final String key = getKey(aliasName, partition);
           ImportPositionEntity importPosition = inflightImportPositions.get(key);
+          registerRecordReaderCompletedGauge(partition, aliasName);
```

Comment:
❓ Why we register this multiple times?

Reply:
What i mean is that we register it now everytime we come into the method 🤔 before it was enough to just do it once the first time.